nano/templates/carrion_spiders.tmpl

```diff
@@ -8,17 +8,43 @@ Used In File(s): code\modules\organs\internal\carrion.dm
 		{{:helper.link('Activate All', 'radiation', {'activate_all' : 1}, null, 'fixedLeft')}}
 	</div>
 </div>
+<div class="item" style="width: 100%;">
+	<div table class="itemContent" style="width: 100%;">
+		{{:helper.link('Activate Group 1', null, {'activate_group_1' : 1}, null)}}
+		{{:helper.link('Activate Group 2', null, {'activate_group_2' : 1}, null)}}
+		{{:helper.link('Activate Group 3', null, {'activate_group_3' : 1}, null)}}
+	</div>
+</table>
+</div>
 <table class='itemContent' style="width: 100%;">
 	<tr>
 		<td><b>Name</b></td>
 		<td><b>Location</b></td>
 		<td><b>Activation</b></td>
+		<td><b>1</b></td>
+		<td><b>2</b></td>
+		<td><b>3</b></td>
 	</tr>
 	{{for data.list_of_spiders}}
 		<tr>
 		<td>{{:value.name}}</td>
 		<td>{{:value.location}}</td>
 		<td>{{:helper.link('activate', 'circle-arrow-s', {'activate_spider' : value.spider})}}</td>
+		{{if value.assigned_group_1}}
+			<td>{{:helper.link('', 'check', {'toggle_group_1' : value.spider})}}</td>
+		{{else}}
+			<td>{{:helper.link('', 'close', {'toggle_group_1' : value.spider})}}</td>
+		{{/if}}
+		{{if value.assigned_group_2}}
+			<td>{{:helper.link('', 'check', {'toggle_group_2' : value.spider})}}</td>
+		{{else}}
+			<td>{{:helper.link('', 'close', {'toggle_group_2' : value.spider})}}</td>
+		{{/if}}
+		{{if value.assigned_group_3}}
+			<td>{{:helper.link('', 'check', {'toggle_group_3' : value.spider})}}</td>
+		{{else}}
+			<td>{{:helper.link('', 'close' , {'toggle_group_3' : value.spider})}}</td>
+		{{/if}}
```

Comment:
You can get rid of the if/elses by using a ternary operator for the button icon.